src/BinSkim.Rules/RuleResources.resx

```diff
@@ -538,4 +538,13 @@ Modules triggering this check were:
   <data name="BA4002_ReportELFCompilerData_Description" xml:space="preserve">
     <value>This rule emits CSV data to the console for every compiler/language/version combination that's observed.</value>
   </data>
+  <data name="BA2026_EnableAdditionalSecurityChecks_Description" xml:space="preserve">
+    <value>/sdl enables a superset of the baseline security checks provided by /GS and overrides /GS-. By default, /sdl is off. /sdl- disables the additional security checks.</value>
+  </data>
+  <data name="BA2026_Error" xml:space="preserve">
```

Comment:
>BA2026_Error [](http://example.com/codeflow?start=14&length=12)

let's make as warning, since this isn't required.

Reply:
sounds good, fixed